maltd-frontend/.env.example

```diff
@@ -1,5 +1,6 @@
+
 REACT_APP_MALTD_API=<base_url_of_backend_api>
 REACT_APP_KEYCLOAK_REDIRECT_URI=<redirect_url>
 REACT_APP_KEYCLOAK_URL=<keycloak_url>
 REACT_APP_KEYCLOAK_REALM=<keycloak_realm>
-REACT_APP_KEYCLOAK_CLIENT_ID=<keycloak_client_id>
+REACT_APP_KEYCLOAK_CLIENT_ID=<keycloak_client_id>
```

Comment:
remove newline on top of this file, and add newline at end of the file please

Reply:
Removed and added lines.